src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -486,6 +486,13 @@ void TorrentHandleImpl::removeUrlSeeds(const QVector<QUrl> &urlSeeds)
         m_session->handleTorrentUrlSeedsRemoved(this, removedUrlSeeds);
 }
 
+void TorrentHandleImpl::clear_peers()
+{
+    #if (LIBTORRENT_VERSION_NUM >= 10207)
+    m_nativeHandle.clear_peers();
+    #endif
```

Comment:
All preprocessing directives must be at the beginning of the line (no indentation).